app/components/workpackages/overview/workpackage-overview.component.html

```diff
@@ -3,13 +3,18 @@
         <button menuToggle>
             <ion-icon name="menu"></ion-icon>
         </button>
-        <ion-title>Workpackages</ion-title>
+        <ion-searchbar [(ngModel)]="_workpackagesSearchString"
+                       debounce="0"
+                       [placeholder]="_context === _contextEnum.BOOKING_WORKPACKAGE_SELECTION ?
+                       'Select Workpackage for Booking' : 'Search for Workpackage'">
+        </ion-searchbar>
     </ion-navbar>
 </ion-header>
-
 <ion-content>
     <ion-list class="slide-in">
-        <ion-card *ngFor="let workpackage of _workpackages" (click)="openWorkpackageDetailComponent(workpackage)">
+        <ion-card *ngFor="let workpackage of _workpackages
+                          | filterListByString: _workpackagesSearchString: ['stringId', 'name']"
```

Comment:
@mamahler: I'm not sure if this is necessary. This could also lead to, having so many results displayed, that it becomes harder to finde the one, you're actually looking for. What do you think @marcelhub @dbabakow?